src/elements/widgets/designerView/designerView.ts

```diff
@@ -282,20 +282,25 @@ export class DesignerView extends BaseCustomWebComponentConstructorAppend implem
     this.designerCanvas.canvasOffset = { x, y };
   }
 
+  
+
   private _onWheel(event: WheelEvent) {
     event.preventDefault();
     if (event.ctrlKey) {
-      let zf = this._designerCanvas.zoomFactor
-      zf += event.deltaY * -0.001; //deltamode = 0
+      let zf = this._designerCanvas.zoomFactor;
+      const wheel = event.deltaY < 0 ? 1 : (-1);
+      zf *= Math.exp(wheel * 0.2);

```

Comment:
@jogibear9988 allgemein fände ich es immer noch sinnvoll, wenn sowas in einer Config festgelegt werden könnte

Reply:
vlt. iwann. Aber bei welchem grafikprogramm stellst du das maus zoom verhalten ein?

Reply:
alles klar